requirements.txt

```
@@ -5,5 +5,5 @@ decorator==4.3
 idna==2.8
 requests==2.21
```

```suggestion
requests==2.22
```
Potentially why the build is failing as requests 2.21 has the following requirement:
"requests 2.21.0 has requirement urllib3<1.25,>=1.21.1"